docs/developer-docs/gas-cost.md

```diff
@@ -4,15 +4,107 @@
 
 The Internet Computer requires computation operations and storage to be supported by cycles. Cycles are generated from the conversion of Internet Computer (ICP) utility tokens.
 
-### The role of the Network Nervous System (NNS) in defining costs
+Canister smart contract computations running on the Internet Computer blockchain are fueled by “cycles”, which play a similar role to “gas” on Ethereum. There are several major differences however. One of the most fundamental differences is that Ethereum leverages “user pays” and the Internet Computer leverages “smart contract pays” (sometimes called “reverse gas”) model. Whereas the Ethereum blockchain requires end users to send payments for the gas smart contracts consume with every transaction, on the Internet Computer, canister smart contracts are pre-charged with cycles, such that contracts effectively pay for their own computation - freeing users from the responsibility.
+
+1 XDR is equal to 1 Trillion cycles. As of November 23, 2022, the exchange rate for 1 XDR = $1.308860, which is used on this page. The exchange rate for USD/XDR may vary and it will impact the conversion rate. You can view XDR exchange rates [here](https://www.imf.org/external/np/fin/data/rms_sdrv.aspx).
```

Comment:
is there a more recent exchange rate we can use? this would be over a year old 😬

Reply:
The cost breakdown chart further on the page uses this rate, so each calculation in that chart would need to be updated. Should we go ahead and update the entire doc with the latest rate?

Reply:
Revised to use the latest XDR rate as of today.